bootstrap/kustomization.yaml

```diff
@@ -1,14 +1,10 @@
 ---
 bases:
-- github.com/argoproj/argo-cd//manifests/cluster-install?ref=v1.5.2
+- github.com/argoproj-labs/argocd-operator//deploy?ref=master
```

Comment:
how "stable" is the `-labs` projects from Argo? Do they by a chance have a release we can use instead of `master`?

Reply:
Are there features/fixes in master that are not in `v0.0.9` that we need?

Reply:
Yepp, we can definitely pin to a release instead of master at this point. The `-labs` project just happens to be the home of the operator project right now and I believe is there because it started as a community project and then was brought in. I'm assuming as it becomes more formal it will move under the main `argoproj` repo.

Reply:
@pabrahamsson @oybed Pinned to v0.0.9